website/hw/hw5.rst

```diff
@@ -53,14 +45,13 @@ Creating your homework repository
 Like previous homeworks, we will provide you with an *invitation URL* that
 will allow you sign up for the homework assignment on GitHub, and which will
 result in the creation of a repository called
-``2021-hw5-GITHUB_USERNAME`` inside our ``uchicago-cmsc22000`` organization
+``hw5-GITHUB_USERNAME`` inside our ``uchicago-cmsc22000`` organization
 on GitHub.
 
 Your repository will be seeded with some files for the homework
 and, more specifically, will contain a ``libgeometry`` directory with an
 updated version of ``libgeometry`` with the refactored ``segment``
-module we asked you to implement in Homework #5.
-
+module we asked you to implement in Homework #2.
 
 
 Task 0: A bit about ``criterion``
```

Comment:
I think we should move the later "Because this homework depends on the
    ``criterion`` library, you should..." up to here, so that students know from the very beginning that they should be working on a CS environment and not their own machines.

Reply:
Good point, I've moved the note up to this point.